lib/zendesk_apps_support/validations/svg.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+require 'loofah'
+
+module ZendeskAppsSupport
+  module Validations
+    module Svg
+      class << self
+        def call(package)
+          errors = []
+          package.svg_files.each do |svg|
+            markup = svg.read
+            clean_markup = Loofah.scrub_xml_document(markup, :prune).to_html
+            filepath = svg.relative_path
+
+            next if clean_markup == markup
```

Comment:
Is the `clean_markup` guaranteed to be unaltered in the case the svg is clean or could Loofah return something slightly reformatted?

Reply:
No, and this is where things could _potentially_ get hairy...

In the case of valid-but-less-than-perfect markup, Loofah will probably alter the contents slightly. I think this is _OK_ though, as the user does get a warning about this happening and should still see their SVG structure preserved and displayed as intended 🤷‍♂️ .

I might add a test case for this though, just for documentation purposes and to recognise that this (for now at least), is intended functionality.